.gitignore

```diff
@@ -350,4 +350,10 @@ MigrationBackup/
 .ionide/
 
 dist
-.angular
\ No newline at end of file
+.angular
+
+.DS_Store
```

Comment:
Can you think of a glob pattern that would ignore all `.DS_Store` files in the repo?